### PR TITLE
Update Kafka, NATS, STAN, RabbitMQ, gRPC, FTP bbes

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/3153

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/ftp-client/ftp_client.bal [hunk 1]
```diff
@@ -12,18 +12,14 @@ public function main() returns error? {
         port: 21,
         auth: {credentials: {username: "user1", password: "pass456"}}
     };
-    ftp:Client clientEp = check new(config);
+    ftp:Client clientEp = check new (config);
 
     // Reads a file from a FTP server for a given file path. In error cases,
```
Comment (praneesha) [inline examples/ftp-client/ftp_client.bal:17]:
```suggestion
    // Reads a file from an FTP server for a given file path. In error cases, 
```

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.description [hunk 2]
```diff
@@ -3,4 +3,4 @@
 // operate when each other sends a sequence of messages using a read-write stream.
 // In such scenarios, the two streams operate independently. Therefore, clients and servers can read and write in any order.<br/><br/>
 // For more information on the underlying module, 
-// see the [GRPC module](https://docs.central.ballerina.io/ballerina/grpc/latest/).
+// see the [GRPC module](https://lib.ballerina.io/ballerina/grpc/latest/).
```
Comment (praneesha) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.description:6]:
```suggestion
// see the [`grpc` module](https://lib.ballerina.io/ballerina/grpc/latest/).
```

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service_client.bal [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
+
 // Creates a gRPC client to interact with the remote server.
-ChatClient ep = check new("http://localhost:9090");
+ChatClient ep = check new ("http://localhost:9090");
 
-public function main () returns error? {
+public function main() returns error? {
     // Executes the RPC call and receives the customized streaming client.
     ChatStreamingClient streamingClient = check ep->chat();
     // Reads server responses in another strand.
```
Comment (praneesha) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service_client.bal:10]:
```suggestion
    // Reads the server responses in another strand.
```

File: examples/grpc-client-basic-auth/grpc_client_basic_auth.description [hunk 4]
```diff
@@ -4,4 +4,4 @@
 // header by passing the `grpc:CredentialsConfig` for the `auth` configuration
 // of the client.<br/><br/>
 // For more information on the underlying module,
-// see the [Auth module](https://docs.central.ballerina.io/ballerina/auth/latest/).
+// see the [Auth module](https://lib.ballerina.io/ballerina/auth/latest).
```
Comment (praneesha) [inline examples/grpc-client-basic-auth/grpc_client_basic_auth.description:7]:
```suggestion
// see the [`auth` module](https://lib.ballerina.io/ballerina/auth/latest).
```

File: examples/grpc-client-basic-auth/grpc_client_basic_auth.md [hunk 5]
```diff
@@ -1,17 +1,34 @@
...
+A client, which is secured with Basic Auth can be used to connect to a secured service.
+
 The client metadata is enriched with the `Authorization: Basic <token>`
-header by passing the `grpc:CredentialsConfig` for the `auth` configuration
-of the client.<br/><br/>
-For more information on the underlying module,
-see the [Auth module](https://docs.central.ballerina.io/ballerina/auth/latest/).
+header by passing the `grpc:CredentialsConfig` for the `auth` configuration of the client.
+
+For more information on the underlying module, see the [Auth module](https://lib.ballerina.io/ballerina/auth/latest).
```
Comment (praneesha) [inline examples/grpc-client-basic-auth/grpc_client_basic_auth.md:8]:
```suggestion
For more information on the underlying module, see the [`auth` module](https://lib.ballerina.io/ballerina/auth/latest).
```

File: examples/grpc-client-basic-auth/grpc_client_basic_auth.md [hunk 6]
```diff
@@ -1,17 +1,34 @@
...
 
 ::: code grpc_client.proto :::
 
+Create a new Protocol Buffers definition file named `grpc_client.proto` and add the service definition to it.
+Run the command below in the Ballerina tools distribution for stub generation.
+
+Once you run the command, `grpc_client_pb.bal` file is generated inside stubs directory.
```
Comment (praneesha) [inline examples/grpc-client-basic-auth/grpc_client_basic_auth.md:15]:
```suggestion
Once you run the command, the `grpc_client_pb.bal` file is generated inside stubs directory.
```

File: examples/grpc-client-bearer-token-auth/grpc_client_bearer_token_auth.md [hunk 7]
```diff
@@ -1,15 +1,32 @@
...
-header by passing the `grpc:BearerTokenConfig` for the `auth` configuration
-of the client.
+header by passing the `grpc:BearerTokenConfig` for the `auth` configuration of the client.
 
 ::: code grpc_client.proto :::
 
+Create a new Protocol Buffers definition file named `grpc_client.proto` and add the service definition to it.
+Run the command below in the Ballerina tools distribution for stub generation.
+
+Once you run the command, `grpc_client_pb.bal` file is generated inside stubs directory.
```
Comment (praneesha) [inline examples/grpc-client-bearer-token-auth/grpc_client_bearer_token_auth.md:13]:
```suggestion
Once you run the command, the `grpc_client_pb.bal` file is generated inside the stubs directory.
```

File: examples/grpc-client-mutual-ssl/grpc_client_mutual_ssl.description [hunk 8]
```diff
@@ -3,4 +3,4 @@
 // verifying the digital certificates. It ensures that both parties are assured
 // of each other's identity.<br/><br/>
 // For more information on the underlying module, 
-// see the [gRPC module](https://docs.central.ballerina.io/ballerina/grpc/latest/).
+// see the [gRPC module](https://lib.ballerina.io/ballerina/grpc/latest).
```
Comment (praneesha) [inline examples/grpc-client-mutual-ssl/grpc_client_mutual_ssl.description:6]:
```suggestion
// see the [`grpc` module](https://lib.ballerina.io/ballerina/grpc/latest).
```

File: examples/grpc-client-mutual-ssl/grpc_client_mutual_ssl.md [hunk 9]
```diff
@@ -2,15 +2,33 @@
 
 Ballerina supports mutual SSL, which is a certificate-based authentication
 process in which two parties (the client and server) authenticate each other by
-verifying the digital certificates. It ensures that both parties are assured
-of each other's identity.<br/><br/>
+verifying the digital certificates. It ensures that both parties are assured of each other's identity.
+
 For more information on the underlying module, 
-see the [gRPC module](https://docs.central.ballerina.io/ballerina/grpc/latest/).
+see the [gRPC module](https://lib.ballerina.io/ballerina/grpc/latest).
```
Comment (praneesha) [inline examples/grpc-client-mutual-ssl/grpc_client_mutual_ssl.md:8]:
```suggestion
see the [`grpc` module](https://lib.ballerina.io/ballerina/grpc/latest).
```

File: examples/grpc-client-mutual-ssl/grpc_client_mutual_ssl.md [hunk 10]
```diff
@@ -2,15 +2,33 @@
...
 
 ::: code grpc_client.proto :::
 
+Create a new Protocol Buffers definition file named `grpc_client.proto` and add the service definition to it.
+Run the command below in the Ballerina tools distribution for stub generation.
+
+Once you run the command, `grpc_client_pb.bal` file is generated inside stubs directory.
```
Comment (praneesha) [inline examples/grpc-client-mutual-ssl/grpc_client_mutual_ssl.md:15]:
```suggestion
Once you run the command, the `grpc_client_pb.bal` file is generated inside the stubs directory.
```

File: examples/grpc-client-mutual-ssl/grpc_client_mutual_ssl.md [hunk 11]
```diff
@@ -2,15 +2,33 @@
...
+
+For more information on how to use the Ballerina Protocol Buffers tool, see the <a href="https://ballerina.io/learn/by-example/proto-to-ballerina.html">Proto To Ballerina</a> example.
+
 ::: out grpc_client.out :::
 
 ::: code grpc_client_mutual_ssl.bal :::
 
-::: out grpc_client_mutual_ssl.out :::
\ No newline at end of file
+Create a Ballerina package.
```
Comment (praneesha) [inline examples/grpc-client-mutual-ssl/grpc_client_mutual_ssl.md:23]:
```suggestion
Create a Ballerina package.
```
We can make these steps numbered bullet points. Check and fix all the occurrences.

File: examples/grpc-client-oauth2-client-credentials-grant-type/grpc_client_oauth2_client_credentials_grant_type.description [hunk 12]
```diff
@@ -4,4 +4,4 @@
 // header by passing the `grpc:OAuth2ClientCredentialsGrantConfig` for the
 // `auth` configuration of the client.<br/><br/>
 // For more information on the underlying module,
-// see the [OAuth2 module](https://docs.central.ballerina.io/ballerina/oauth2/latest/).
+// see the [OAuth2 module](https://lib.ballerina.io/ballerina/oauth2/latest).
```
Comment (praneesha) [inline examples/grpc-client-oauth2-client-credentials-grant-type/grpc_client_oauth2_client_credentials_grant_type.description:7]:
```suggestion
// see the [`oauth2` module](https://lib.ballerina.io/ballerina/oauth2/latest).
```

File: examples/grpc-client-oauth2-client-credentials-grant-type/grpc_client_oauth2_client_credentials_grant_type.md [hunk 13]
```diff
@@ -1,17 +1,37 @@
...
+can be used to connect to a secured service.
+
 The client metadata is enriched with the `Authorization: Bearer <token>`
 header by passing the `grpc:OAuth2ClientCredentialsGrantConfig` for the
-`auth` configuration of the client.<br/><br/>
+`auth` configuration of the client.
+
 For more information on the underlying module,
-see the [OAuth2 module](https://docs.central.ballerina.io/ballerina/oauth2/latest/).
+see the [OAuth2 module](https://lib.ballerina.io/ballerina/oauth2/latest).
```
Comment (praneesha) [inline examples/grpc-client-oauth2-client-credentials-grant-type/grpc_client_oauth2_client_credentials_grant_type.md:11]:
```suggestion
see the [OAuth2 module](https://lib.ballerina.io/ballerina/oauth2/latest).
```
Same as above. Please check and fix all occurrences.

File: examples/grpc-client-oauth2-client-credentials-grant-type/grpc_client_oauth2_client_credentials_grant_type.md [hunk 14]
```diff
@@ -1,17 +1,37 @@
...
 
 ::: code grpc_client.proto :::
 
+Create a new Protocol Buffers definition file named `grpc_client.proto` and add the service definition to it.
+Run the command below in the Ballerina tools distribution for stub generation.
+
+Once you run the command, `grpc_client_pb.bal` file is generated inside stubs directory.
```
Comment (praneesha) [inline examples/grpc-client-oauth2-client-credentials-grant-type/grpc_client_oauth2_client_credentials_grant_type.md:18]:
```suggestion
Once you run the command, the `grpc_client_pb.bal` file is generated inside the stubs directory.
```
Please check and fix all the occurrences.

**Comment by praneesha** [comment]
@dilanSachi - The above review comments need to be addressed in all the BBEs in this PR.

**Comment by dilanSachi** [comment]
> @dilanSachi - The above review comments need to be addressed in all the BBEs in this PR.

Updated

**Pull request merged**